### PR TITLE
Setup Celery as part of Django setup

### PR DESCRIPTION
**Pull request by millerdev**
In addition to a bit of cleanup, this allows `manage.py` to import things from `corehq` without triggering Celery initialization as a side effect.

:blowfish: Review by commit.

### Safety story

Tested on staging. Web workers and celery processes appear to be working normally.

### Automated test coverage

No new test coverage.

### QA Plan

No QA.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -115,25 +113,13 @@ def run_patches():
...
     django.setup = _setup_once
 
 
-def patch_pickle():
-    """Patch pickle to support protocol 5
-
-    Remove after upgrading to Python 3.8+
-    """
-    import pickle
-    if pickle.HIGHEST_PROTOCOL < 5:
```
Comment (dannyroberts) [inline manage.py:132]: Will we need this function again the next time a python upgrade includes a new highest pickle protocol?

**Review by dannyroberts — APPROVED** [review]

**Comment by dannyroberts** [comment]
Were the tests manually cancelled, or did something happen?

File: manage.py (hunk 1 above)
Comment (millerdev) [reply manage.py:132]: If we do I know where to dig it up. I hope we are upgraded to a new version of Celery with better behavior by the time we would need it.

**Comment by millerdev** [comment]
> Were the tests manually cancelled, or did something happen?

They were manually cancelled. The problem was caused by Celery tasks being imported before the Celery app was initialized, causing them to become associated with an automatically created "default" Celery app, which caused `corehq.apps.users.tests.test_signals:TestUserSignals.test_webuser_save` to hang in `corehq/apps/analytics/signals.py:user_save_callback` on `identify.delay(couch_user.username, properties)`. I think the fact that the default Celery app had `task_always_eager == False` was related to the hang, but I didn't dig into the precise reason once I realized that the default app was being used when it should not have been.

**Review by snopoke — APPROVED** [review]

**Review by dannyroberts — APPROVED** [review]

**Pull request merged**